src/moscot/backends/ott/solver.py

```diff
@@ -43,14 +44,19 @@ def __init__(self, jit: bool = True):
         self._solver: Optional[OTTSolver_t] = None
         self._problem: Optional[OTTProblem_t] = None
         self._jit = jit
+        self._a: Optional[jnp.ndarray] = None
+        self._b: Optional[jnp.ndarray] = None
+        self._requires_graph_output = False  # resolve which output to use, OTTOutput v. GraphOTTOutput
```

Comment:
Remove this please, not necessary - can be done via `isinstance` check later, would do
```def _solver_output_type(self, prob: ....) -> Union[Type[...], Type[...]]:
   # if a geodesic, require graph output, otherwise normal.
```